CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 ## vNext (Month Day Year)
 
 **New This Week**
-- :bug: Bugfix: Fix parsing bug where whitespace was stripped when parsing XML (#590)
+- :tada: Release of all* remaining AWS services! This release includes all remaining AWS services with the exception of three
```

Comment:
You have the asterisk, but I think it might be clearer if you just directly say, "Release of all but three remaining AWS services! Glacier, IOT Device Gateway, and Transcribe Streaming still need additional work and are not included."

Reply:
Rust always talks about future editions. How about: will be available in a future release? Sounds more positive.